atlas_core/core.py

```diff
@@ -4,3 +4,10 @@
 
 #: Flask-SQLAlchemy db object
 db = SQLAlchemy()
+
+# Need these to be generated in order to drop them when loading
+db.metadata.naming_convention = {
+    "ix": "%(table_name)s_%(column_0_name)s_idx",
+    "fk": "%(table_name)s_%(column_0_name)s_fkey",
+    "pk": "%(table_name)s_pkey",
+}
```

Comment:
Probably worth adding in the unique constraint and check constraint ones too, from here: http://docs.sqlalchemy.org/en/latest/core/constraints.html#configuring-constraint-naming-conventions